### DISCUSSION TITLE
Which version to ship with Coq Platform 2023.10?

### DISCUSSION THREAD

**Discussion by benediktahrens** [UniMath/UniMath#1789] [General]
See https://github.com/UniMath/UniMath/issues/1788.

Is it desirable to ship a more recent version than one from April 2023?

**Comment by nmvdw** [comment]
What do we have to do the ship the most recent version? Create a new tag? Or more?

**Reply by benediktahrens** [reply]
1. Create a new tag
2. Create a new opam package pointing to this tag
3. Upload that opam package somewhere

I don't know how to do 2. and 3. :-(

**Reply by nmvdw** [reply]
I also don't know how to do 2 and 3

A newer version would be preferred (it's time anyway to create a new tag).
